app/controllers/atmosphere/api/v1/virtual_machine_flavors_controller.rb

```diff
@@ -19,7 +19,16 @@ def index
           cs_id = params[:compute_site_id]
 
           if optimizer_query?
-            tmpls = VirtualMachineTemplate.active.on_active_tenant
+            tmpls = VirtualMachineTemplate
+              .active
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.